.github/workflows/alpha.yaml

```diff
@@ -230,7 +230,7 @@ jobs:
         # GITHUB_REPOSITORY:
         # GITHUB_WORKSPACE:
         SCOPE_DSN_PUBLIC: ""
-      run: export $(cat .image.env | sed 's/#.*//g' | xargs) && make test build kots
+      run: export $(cat .image.env | sed 's/#.*//g' | xargs) && make ci-test kots build
```

Comment:
is it ok to switch this everywhere to ci-test to unify builds between workflows?

Reply:
yeah, `ci-test` is a superset of `test`